### ISSUE TITLE
[question] retrain model after loading checkpoint

### ISSUE THREAD

**Issue by wujiren** [DLR-RM/stable-baselines3#51]
**Problem describe**
For some reasons, our model is trained in steps. After each step checkpoint, we decide  whether change some setting or not, and then load the model for further training. Sometimes, even we did not change anything in the model, the training performance (not the predict performance) after loaded had a gap with the previous performance in training. The awkward thing is that the gap seems random, we are not sure where the problem comes from and it is hard to show the gap in model. We have checked the params in the policy, and sure that the params is correctly loaded every time, the learning rate/clip_range etc is a default constant value, so we guess that the problem might comes from the Optimizer because the Optimize is not saved in the save method.
So if you have any option about the retrain procedure, please give us some guidance, and should we have to save the Optimizer in the checkpoint.
Best wishes.

**Comment by araffin** [comment]
Hello,
Please fill the issue template completely.
Duplicate of https://github.com/DLR-RM/stable-baselines3/issues/29

The optimizer is saved.
